test/common/odp_cunit_common.c

```diff
@@ -292,7 +326,9 @@ static int cunit_register_suites(odp_suiteinfo_t testsuites[])
 		if (sinfo->term_func)
 			term_func = sinfo->term_func;
 
-		suite = CU_add_suite(sinfo->name, _cunit_suite_init, term_func);
+		suite = CU_add_suite_with_setup_and_teardown(sinfo->name, _cunit_suite_init,
```

Comment:
`CU_add_suite_with_setup_and_teardown()` is not available on CentOS 7, which causes the CI to fail.

Reply:
Okay. The CUnit version on Centos 7 could be older. Let me see if there is a way to workaround the version compatibiltiy.

Reply:
Hi @ashwinyes, I just updated the CentOS 7 CI image and the build should now work. Please rebase this PR on top of the latest master.